test/functional/wallet_labels.py

```diff
@@ -25,6 +25,40 @@ def set_test_params(self):
     def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
 
+    def invalid_label_name_test(self):
+        node = self.nodes[0]
+        address = node.getnewaddress()
+        pubkey = node.getaddressinfo(address)['pubkey']
+        rpc_calls = [
+            [node.getnewaddress],
+            [node.setlabel, address],
+            [node.getaddressesbylabel],
+            [node.importpubkey, pubkey],
+            [node.addmultisigaddress, 1, [pubkey]],
+            [node.getreceivedbylabel],
```

Comment:
Looks like listsinceblock ought to be added to this test if #25934 is merged.

Reply:
answered here: https://github.com/bitcoin/bitcoin/pull/25934#discussion_r984481844